nistats/hemodynamic_models.py

```diff
@@ -482,16 +482,22 @@ def compute_regressor(exp_condition, hrf_model, frame_times, con_id='cond',
      - 'glover + derivative': the Glover hrf + time derivative (2 regressors)
      - 'glover + derivative + dispersion': idem + dispersion derivative
                                         (3 regressors)
-    'fir': finite impulse response basis, a set of delayed dirac models
-           with arbitrary length. This one currently assumes regularly spaced
-           frame times (i.e. fixed time of repetition).
+    'fir': list or array,
+        finite impulse response basis, a set of delayed dirac models.
     
     It is expected that spm standard and Glover model would not yield
     large differences in most cases.
 
     In case of glover and spm models, the derived regressors are
     orthogonalized wrt the main one.
     """
+    # fir_delays should be integers
+    if fir_delays:
```

Comment:
Seems to trigger an error in Travis. I think it should be 
```suggestion
    if fir_delays is not None:
```